### DISCUSSION TITLE
NER - improve recall

### DISCUSSION THREAD

**Discussion by XBeg9** [explosion/spaCy#10127] [Help: Coding & Implementations] [answered]
I know there is no confidence scores for NER component, which is very strange, but is there a way to control recall for this component? like setting threshold settings, recall is very important for me comparing with precision.

**Comment by XBeg9** [comment]
One of the things that I have tried and got a bit of improvement is to train my own tok2vec model using this: https://spacy.io/usage/embeddings-transformers#pretraining

Anything else where should look into, would love to get any help! Thanks

**Comment by polm** [comment, marked as answer]
No, we don't really have any parameters you can fiddle with in the NER component to directly influence this. It's from 2019 but you can see [a comment about how to approach this here](https://github.com/explosion/spaCy/issues/3872#issuecomment-509788710). It's worth keeping in mind that we don't hide these parameters out of forgetfulness - the NER architecture we use is a bit unusual (it's the same as our parser) and it's not always clear how parameters common in other systems map to it.

What you can do is use a spancat instead and use a lower than default threshold.
